docs/target-compatibility.md

```diff
@@ -299,7 +299,7 @@ Currently support is available in D3D12 via NVAPI, and for Vulkan via the [GL_NV
 
 Slang has preliminary support for `debugBreak()` intrinsic. With the appropriate tooling, when `debugBreak` is hit it will cause execution to halt and display in the attached debugger.
 
-Currently this is supported in all targets except HLSL. Note that on some targets if there isn't an appropriate debugging environment the debugBreak might cause execution to fail or potentially it is ignored.
+Tihs is not supported on HLSL, GLSL, SPIR-V or Metal backends. Note that on some targets if there isn't an appropriate debugging environment the debugBreak might cause execution to fail or potentially it is ignored.
```

Comment:
Typo: "Tihs" -> "This".

Also, I would reformulate:
"Note that on some targets, if there isn't an appropriate debugging environment, `debugBreak` might be ignored or else cause execution to fail."